public/content/roadmap/fusaka/index.md

```diff
@@ -44,11 +42,14 @@ These coordinated upgrades generally include a lot of changes, require a lot of
 
 Blob parameter only forks can be set by clients, similarly to other configuration like gas limit. Between major Ethereum upgrades, clients can agree to increase the `target` and `max` blobs to e.g. 9 and 12 and then node operators will update to take part in that tiny fork. These blob parameter only forks can be configured at any time.
 
-#### Blob base-fee bounded by execution costs {#blob-base-fee-bounded-by-execution-costs}
+When blobs were first added to the network in the Dencun upgrade, the target was 3 - that was increased to 6 in Pectra and, after Fusaka, that can now be increased at a sustainable rate independently of these major network upgrades.
 
-Specification: https://eips.ethereum.org/EIPS/eip-7918
+[ IMAGE HERE, ETH.ORG TEAM PLEASE HELP ATTACH: https://drive.google.com/file/d/1U24kK-XbZ_l6UWwDO_2-0Mxn4kSkEwVe/view?usp=sharing ]
```

Comment:
Will upload the image to this directory... We need to add alt text for this though, so if you could update the following with a description:

```suggestion
![todo: replace this with description of diagram for screen readers](./average-blob-count-per-block.webp)

```